test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -506,11 +588,11 @@ def forward(self, x):
             additional_test_inputs=[((x2,),)],
         )
 
-    @pytorch_test_common.xfail(
```

Comment:
Generally, xfail is preferred than skip. To compromise the new test, this one goes to skip.

Reply:
moved to pytorch_test_common.py and used xfail instead of skip